Cargo.lock

```diff
@@ -1510,9 +1510,9 @@ dependencies = [
  "env_logger",
  "fuel-asm",
  "fuel-storage",
- "fuel-tx",
+ "fuel-tx 0.1.0",
```

Comment:
it seems like we're getting mismatched versions of fuel-tx somehow

Reply:
Probably some library that uses `fuel-tx` or `fuel-vm` is not updated, will check.

Reply:
![fueltx_01](https://user-images.githubusercontent.com/13179220/149619331-58fd2f04-fad4-4579-9bad-a8dfa1bb9799.png)

It is used by example project `simple-wasm` that imports `fuels-abigen-macro`->`forc`->`fuel-gql-client`. 
To to do the update, we need to merge `fuel-gql-client` and update version to `forc`/`fuel-abigen-macro`

Reply:
Given how messy this is, and how the example project is more of an integration thing, should it simply be removed from the workspace?

Reply:
Seems resonable to be removed or moved to somewhere else if still needed (not sure where).

Reply:
#140